docs/manage-fileset-metadata-using-gravitino.md

```diff
@@ -0,0 +1,422 @@
+```
+
+</TabItem>
+<TabItem value="java" label="Java">
+
+```java
+GravitinoClient gravitinoClient = GravitinoClient
+    .builder("http://127.0.0.1:8090")
+    .build();
+
+// Assuming you have just created a metalake named `metalake`
+GravitinoMetaLake gravitinoMetaLake =
+    gravitinoClient.loadMetalake(NameIdentifier.of("metalake"));
+
+// Assuming you have just created a Hadoop catalog named `catalog`
+Catalog catalog = gravitinoMetaLake.loadCatalog(NameIdentifier.of("metalake", "catalog"));
+
+SupportsSchemas supportsSchemas = catalog.asSchemas();
+
+Map<String, String> schemaProperties = ImmutableMap.<String, String>builder()
+    // Property "location" is optional, if specified all the managed fileset without
+    // specifying storage location will be stored under this location.
+    .put("location", "file:/tmp/root/schema")
+    .build();
+Schema schema = supportsSchemas.createSchema(
+    NameIdentifier.of("metalake", "catalog", "schema"),
+    "This is a schema",
+    schemaProperties
+);
+// ...
+```
+
+</TabItem>
+</Tabs>
+
+Currently, Gravitino supports the following schema property:
+
+| Catalog provider    | Schema property                                                              |
+|---------------------|------------------------------------------------------------------------------|
+| `hadoop`            | [Hadoop schema property](./hadoop-catalog.md#schema-properties)              |
+
+### Load a schema
+
+Please refer to [Load a schema](./manage-relational-metadata-using-gravitino.md#load-a-schema)
+from relational catalog for more details. For fileset catalog, the schema load operation is the
+same.
+
+### Alter a schema
+
+Please refer to [Alter a schema](./manage-relational-metadata-using-gravitino.md#alter-a-schema)
+from relational catalog for more details. For fileset catalog, the schema alter operation is the
+same.
+
+### Drop a schema
+
+Please refer to [Drop a schema](./manage-relational-metadata-using-gravitino.md#drop-a-schema)
+from relational catalog for more details. For fileset catalog, the schema drop operation is the
+same.
+
+Note that the drop operation will also remove all the filesets as well as the managed files
+under this schema path if `cascade` is set to `true`.
+
+### List all schemas under a catalog
+
+Please refer to [List all schemas under a catalog](./manage-relational-metadata-using-gravitino.md#list-all-schemas-under-a-catalog)
+from relational catalog for more details. For fileset catalog, the schema list operation is the
+same.
+
+## Fileset operations
+
+:::tip
+ - Users should create a metalake, a catalog and a schema before creating a fileset.
+ - Current Gravitino only supports managing Hadoop Compatible File System (HCFS) locations.
+:::
+
+### Create a fileset
+
+You can create a fileset by sending a `POST` request to the `/api/metalakes/{metalake_name}
+/catalogs/{catalog_name}/schemas/{schema_name}/files` endpoint or just use the Gravitino Java
```

Comment:
`/api/metalakes/{metalake_name}
/catalogs/{catalog_name}/schemas/{schema_name}/files`
--
Should it be `metalakes/{metalake}/catalogs/{catalog}/schemas/{schema}/filesets`? https://github.com/datastrato/gravitino/blob/15443039113ee3b8a14591feb262cccf6431f952/server/src/main/java/com/datastrato/gravitino/server/web/rest/FilesetOperations.java#L40